src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.servlets;
+ 
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+ 
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+ 
+import com.google.sps.data.Comment;
+ 
+import java.util.List;
+import java.util.ArrayList;
+ 
+import com.google.gson.Gson;
+ 
+import java.io.IOException;
+ 
+/*
+ * This servlet takes care of the comment functionality in the 
+ * 1Deed1Day webapp, allows users to input comments and also displays
+ * all the previous comments
+ */
+@WebServlet("/comments")
+public class CommentServlet extends HttpServlet {
+ 
+    private List<Comment> comments = new ArrayList<>();
```

Comment:
what would be the benefit of having `comments` as a member variable? Since it's only used in `doGet()` it can be a local variable inside that method. This way, you don't have to clear the list each time before processing it

Reply:
Makes sense!